build/data/repository-config.ts

```diff
@@ -68,7 +68,9 @@ export const astroRepositoryConfig = {
             'src/**/*.mdx',
             'src/**/*.js',
             'src/**/*.ts',
-            'public/**/*.js'
+            'public/**/*.js',
+            'node_modules/choco-astro/src/**/*.astro',
+            'node_modules/choco-astro/src/**/*.ts'
```

Comment:
@steviecoaster These are the main changes. Most of everything else are auto generated files as a result of updating choco-theme to a new version.